hedera-node/hedera-config/src/main/java/com/hedera/node/config/data/ContractsConfig.java

```diff
@@ -46,7 +46,7 @@ public record ContractsConfig(
         @ConfigProperty(defaultValue = "15000000") @NetworkProperty long maxGasPerSec,
         @ConfigProperty(value = "maxKvPairs.aggregate", defaultValue = "500000000") @NetworkProperty
                 long maxKvPairsAggregate,
-        @ConfigProperty(value = "maxKvPairs.individual", defaultValue = "163840") @NetworkProperty
+        @ConfigProperty(value = "maxKvPairs.individual", defaultValue = "16384000") @NetworkProperty
```

Comment:
This might interact poorly with the default on line 31, which appears to be _possibly_ incorrect... (it has `0til100M,2000til450M`, which might be intended as `0til100M,200Mtil450M` (note 200M instead of 2000, not entirely sure)

Reply:
According to the comments [here](https://github.com/hashgraph/hedera-services/blob/develop/hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/fees/charging/ContractStoragePriceTiers.java#L54), the value in line 31 looks to be correct and probably not related to configuration value that I changed.